### PR TITLE
Floret custom dc

### PR DESCRIPTION
**Pull request by dwnoble**
Screenshot:

Screenshot 2023-10-02 at 4 03 58 PM

### CODE REVIEW

File: server/app_env/floret.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+class Config(_base.Config):
+  CUSTOM = True
+  SHOW_TOPIC = True
+  NAME = "FLORET Data Commons"
+  LOG_QUERY = True
+  LOGO_PATH = "/custom_dc/floret/logo.svg"
+  OVERRIDE_CSS_PATH = '/custom_dc/floret/overrides.css'
+  SHOW_DISASTER = False
+  USE_PALM = True
```
Comment (shifucun) [inline server/app_env/floret.py:27]: USE_PALM = False? Since they don't have NL now

File: server/templates/custom_dc/floret/base.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
+{#-
+  Copyright 2020 Google LLC
```
Comment (shifucun) [inline server/templates/custom_dc/floret/base.html:2]: 2023

File: server/templates/custom_dc/floret/base.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+      {{ ("send_page_view: false,"|safe) if manual_ga_pageview else "" }}
+    });
+  </script>
+  {% endif %}
+
+  <title>{{ title }}{{' - ' if title else ''}}Data Commons</title>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+  {% if config['HIDE_REVAMP_CHANGES'] %}
```
Comment (shifucun) [inline server/templates/custom_dc/floret/base.html:60]: maybe remove these condition check here?

The base.html is only used for this instance, can could have the minimal needed content.

File: server/templates/custom_dc/floret/homepage.html [hunk 4]
```diff
@@ -0,0 +1,228 @@
...
+ 
+ {%- extends BASE_HTML -%}
+ 
+ {% set main_id = 'homepage' %}
+ {% set page_id = 'page-homepage' %}
+ 
+ {% block head %}
+ <link rel="stylesheet" href={{url_for('static', filename='css/homepage.min.css', t=config['VERSION'])}}>
+ <script src={{url_for('static', filename='homepage.js', t=config['VERSION'])}} async></script>
+ <script src="http://localhost:8080/datacommons.js"></script>
```
Comment (shifucun) [inline server/templates/custom_dc/floret/homepage.html:25]: is this localhost?

File: server/app_env/floret.py (hunk 1 above)
Comment (dwnoble) [reply server/app_env/floret.py:27]: Done!

File: server/templates/custom_dc/floret/base.html (hunk 2 above)
Comment (dwnoble) [reply server/templates/custom_dc/floret/base.html:2]: Done!

File: server/templates/custom_dc/floret/base.html (hunk 3 above)
Comment (dwnoble) [reply server/templates/custom_dc/floret/base.html:60]: Good point-removed!

File: server/templates/custom_dc/floret/homepage.html (hunk 4 above)
Comment (dwnoble) [reply server/templates/custom_dc/floret/homepage.html:25]: Fixed-changed to dev.datacommons.org for now (until the next prod push)

**Review by shifucun — APPROVED** [review]
thanks for the update!

**Review by keyurva — APPROVED** [review]
Thanks Dan!

**Pull request merged**